ui/src/app/streams/streams/deployment-properties/deployment-properties.component.ts

```diff
@@ -76,8 +77,11 @@ export class DeploymentPropertiesComponent implements OnInit, OnDestroy {
       if (this.stream.deploymentProperties.platformName) {
         this.deploymentPlatform.setValue(this.stream.deploymentProperties.platformName);
       }
+      if (this.stream.deploymentProperties[keyPlatform]) {
+        this.deploymentPlatform.setValue(this.stream.deploymentProperties[keyPlatform])
```

Comment:
with this addition, what exactly is above `this.deploymentPlatform.setValue(this.stream.deploymentProperties.platformName)` supposed to do?

Reply:
I didn't want to remove this line but I think it's dead code...